src/main/scala/com/databricks/labs/overwatch/ParamDeserializer.scala

```diff
@@ -183,11 +183,11 @@ class ParamDeserializer() extends StdDeserializer[OverwatchParams](classOf[Overw
 
     val apiEnvConfig = if (masterNode.has("apiEnvConfig")) {
       Some(ApiEnvConfig(
-        getOptionInt(masterNode, "apiEnvConfig.successBatchSize").getOrElse(200),
-        getOptionInt(masterNode, "apiEnvConfig.errorBatchSize").getOrElse(500),
-        getOptionBoolean(masterNode, "apiEnvConfig.enableUnsafeSSL").getOrElse(false),
-        getOptionInt(masterNode, "apiEnvConfig.threadPoolSize").getOrElse(4),
-        getOptionLong(masterNode, "apiEnvConfig.apiWaitingTime").getOrElse(300000),
+        Some(getOptionInt(masterNode, "apiEnvConfig.successBatchSize").getOrElse(200)),
+        Some(getOptionInt(masterNode, "apiEnvConfig.errorBatchSize").getOrElse(500)),
+        Some(getOptionBoolean(masterNode, "apiEnvConfig.enableUnsafeSSL").getOrElse(false)),
+        Some(getOptionInt(masterNode, "apiEnvConfig.threadPoolSize").getOrElse(4)),
+        Some(getOptionLong(masterNode, "apiEnvConfig.apiWaitingTime").getOrElse(300000)),
```

Comment:
@Sriram-databricks -- I understand that you asked @souravbaner-da to make this change but I don't quite understand why this is necessary. I think this may go back to a larger conversation of how we're initializing these values but I'd like some detail here.